backend/src/api/integration/helpers/redditValidator.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+import Error400 from '../../../errors/Error400'
+import Permissions from '../../../security/permissions'
+import PermissionChecker from '../../../services/user/permissionChecker'
+import track from '../../../segment/telemetryTrack'
+
+export default async (req, res) => {
+  new PermissionChecker(req).validateHasAny([
+    Permissions.values.integrationCreate,
+    Permissions.values.integrationEdit,
+  ])
+
+  if (req.query.subreddit) {
+    try {
+      const result = await axios.get(
+        `https://www.reddit.com/r/${req.query.subreddit}/new.json?limit=1`,
+      )
+      if (
+        result.status === 200 &&
+        result.data.data.children &&
+        result.data.data.children.length > 0
+      ) {
+        track(req, 'Reddit: subreddit input', {
```

Comment:
Are we sure we want track to these (both the valid and not valid)?
What kind of insights are we hoping to get?

Reply:
We would have the option to implement fetching the subreddits that people own. I am hoping this will give us a sense of whether people are struggling with the setup, to see if we should implement that.